spec/views/hyrax/dashboard/collections/_work_action_menu.html.erb_spec.rb

```diff
@@ -2,11 +2,13 @@
 
 RSpec.describe 'hyrax/dashboard/collections/_work_action_menu.html.erb', type: :view do
   let(:item) { stub_model(GenericWork) }
-  let(:collection) { stub_model(Collection) }
+  let(:collection) { stub_model(Collection, id: 'xyz123z4', title: ["Make Collections Great Again"]) }
+  let(:change_set) { Hyrax::CollectionChangeSet.new(collection) }
 
   before do
     allow(view).to receive(:display_trophy_link)
     assign(:collection, collection)
```

Comment:
I think you can remove this?

Reply:
Following on to Justin's comment do you need to assign the change set here instead of collection?